crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -211,9 +320,6 @@ impl<'a> MoveTestAdapter<'a> for SuiTestAdapter {
         self.default_syntax
     }
     async fn cleanup_resources(&mut self) -> anyhow::Result<()> {
```

Comment:
I suspect the only reason we added this part of the interface was to clean up the executor cluster, so if that is no longer needed, we can get rid of this entirely.